src/components/CaseDetails/styles.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+// case detail
+export const CaseDisplay = styled.aside`
+  position: relative;
+  width: 100%;
+`;
+
+export const CaseTitle = styled.h1`
+  margin: 0.4rem 4rem;
+`;
+
+export const CaseInterestContainer = styled.div`
+  position: sticky;
+  top: 4rem;
+  width: 95%;
+  border-radius: 10px;
```

Comment:
Shall we use `rem` here?

Reply:
resolved this in slack DMs with sauhard! this is what he said:

> i would rec using px
> reason is rem depends on root font size and border widths have no relation to font size
> and the variance is border width across screen size is pretty limited

Reply:
so let's keep this as is

Reply:
I think for `border-radius` we will still use `rem`, only `px` for `border-width`

Reply:
@pragyakallanagoudar are you fine with going along with this change? I think it makes sense to change `border-radius` to `rem`